Typo fixes(ExpandableVariables.md)

Luro223 opened the pull request.

Luro223 committed:
Typo Fix(ExpandableVariables.md)

isaak654 committed:
Update ExpandableVariables.md

isaak654 committed:
Update ExpandableVariables.md

isaak654 reviewed (commented):
Please review a3ec0fbf57325da0ba8f16415d3bbd1bf1d5a5c1 (thumbs up for approval).

isaak654 committed:
Merge branch 'main' into patch-18

isaak654 committed:
Update ExpandableVariables.md

Luro223 committed:
Update ExpandableVariables.md

will cause Sandboxie to not look in the registry at all >> will skip the registry addressing for Sandboxie

Luro223 committed:
Update ExpandableVariables.md

isaak654 commented on Content/ExpandableVariables.md:
````diff
@@ -74,6 +74,6 @@ For example:
 ```
 
-Note that "Ovr."-style overrides (described above) will cause Sandboxie to not look in the registry at all. On the other hand, Sandboxie only considers "Reg."-style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never appl ...
+Note that "Ovr." style overrides (described above) will skip the registry addressing for Sandboxie. On the other hand, Sandboxie only checks "Reg." style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never apply.
````
The term "addressing" seems a bit too technical, is there a simpler one?

Luro223 replied:
will force/make/cause Sandboxie (to) ignore the registry (lookup)

isaak654 replied:
https://github.com/sandboxie-plus/sandboxie-docs/pull/79#pullrequestreview-1121933400

isaak654 commented on Content/ExpandableVariables.md:
````diff
@@ -74,6 +74,6 @@ For example:
 ```
 
-Note that "Ovr."-style overrides (described above) will cause Sandboxie to not look in the registry at all. On the other hand, Sandboxie only considers "Reg."-style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never appl ...
+Note that "Ovr." style overrides (described above) will skip the registry addressing for Sandboxie. On the other hand, Sandboxie only checks "Reg." style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never apply.
````
```suggestion
Note that "Ovr." style overrides (described above) will cause Sandboxie to ignore the registry. On the other hand, Sandboxie only checks "Reg." style fallbacks if the expanded variable cannot be found in the registry. This means that if both Ovr.X and Reg.X are specified for the same variable X, the Ovr.X form will always apply when X is expanded, and the Reg.X form will never apply.
```

Luro223 committed:
Update Content/ExpandableVariables.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>

Luro223 committed:
Update ExpandableVariables.md

Luro223 committed:
Update ExpandableVariables.md

isaak654 commented on Content/ExpandableVariables.md:
```diff
@@ -30,15 +30,15 @@ The following table lists the variables that Sandboxie recognizes.
 ### Overriding Variables
 
-Any of the variables in the table above, including any of the [Shell Folders](ShellFolders.md) variable and any template variables, can be overridden by the [Sandboxie Ini](SandboxieIni.md) configuration file. To override a variable, add a setting with an **Ovr.** prefix.
+Any of the variables in the table above, including any of the [Shell Folders](ShellFolders.md) variables and any of the template variables, can be overridden by the [Sandboxie Ini](SandboxieIni.md) configuration file. To override a variable, add a parameter prefixed with **Ovr.**.
```
```suggestion
Any of the variables in the table above, including the [Shell Folders](ShellFolders.md) and template variables, can be overridden by the [Sandboxie Ini](SandboxieIni.md) configuration file. To override a variable, add a parameter prefixed with **Ovr.**.
```

Luro223 committed:
Update Content/ExpandableVariables.md

Co-authored-by: isaak654 <isaak654@users.noreply.github.com>
